package.json

```diff
@@ -60,5 +68,8 @@
     "node": ">=12.0.0",
     "npm": ">=6",
     "yarn": "please use npm"
+  },
+  "resolutions": {
+    "lodash": "^4.17.19"
```

Comment:
I was getting an audit failure because Vorpal was bringing in a slightly older version of lodash (4.5 I believe) which had 2 high security issues. Vorpal project itself has an open PR to update the dependency. But since it's not merged or released the audit was failing so this seemed like a reasonable way to override the dependency to the latest version of lodash.